lib/enkf/analysis_config.cpp

```diff
@@ -604,6 +606,53 @@ void analysis_config_free(analysis_config_type * config) {
   delete config;
 }
 
+analysis_config_type * analysis_config_alloc_full(
+                            double alpha,
+                            bool merge_observations,
+                            bool rerun,
+                            int rerun_start,
+                            const char * log_path,
+                            double std_cutoff,
+                            bool stop_long_running,
+                            bool single_node_update,
+                            bool std_scale_correlated_obs,
+                            double global_std_scaling,
+                            bool store_PC,
+                            const char * PC_filename,
+                            const char * PC_path,
+                            int max_runtime,
+                            int min_realisations
+) {
+  analysis_config_type * config = new analysis_config_type();
+  UTIL_TYPE_ID_INIT( config , ANALYSIS_CONFIG_TYPE_ID );
+
+  config->update_settings           = config_settings_alloc( UPDATE_SETTING_KEY );
+  config_settings_add_double_setting(config->update_settings, UPDATE_OVERLAP_KEY , alpha );
+  config_settings_add_double_setting(config->update_settings, UPDATE_STD_CUTOFF_KEY, std_cutoff );
+
+  analysis_config_set_merge_observations( config       , merge_observations );
```

Comment:
Simple setting functions can be reduced to setting the property directly

Reply:
Done